dds/DCPS/Serializer.cpp

```diff
@@ -546,6 +546,7 @@ bool Serializer::read_parameter_id(unsigned& id, size_t& size)
     }
 
     // TODO(iguessthislldo): handle PID flags
+    must_understand = false; // Placeholder
```

Comment:
We can set `must_understand` to the second bit of `pid` in case it's not PID_EXTENDED.

Reply:
Yes, but we're deferring further XCDR1 support for this iteration.

Reply:
Gotcha. Also, I haven't seen changes for `write_parameter_id`. Is this function supposed to get an input parameter for must_understand from the caller? Or does it need to get that flag itself?

Reply:
Writer side support is also deferred. https://trello.com/c/EtGCQW8X/86-xcdr-parameter-list-flags

Reply:
I see.